examples/shared_samples/logical_interconnect_group.rb

```diff
@@ -26,7 +26,10 @@
 # Resource Class used in this sample
 lig_class = OneviewSDK.resource_named('LogicalInterconnectGroup', @client.api_version)
```

Comment:
To create a lig_class for synergy variant, you have to explicitly pass the variant as 3rd argument. Otherwise it will consider C7000 as variant by default.
```
variant = 'Synergy'
lig_class = OneviewSDK.resource_named('LogicalInterconnectGroup', @client.api_version, variant)
```